Mathlib/Data/Vector/Defs.lean

```diff
@@ -83,14 +104,32 @@ theorem cons_head_tail : ∀ v : Vector α (succ n), cons (head v) (tail v) = v
 def toList (v : Vector α n) : List α :=
   v.1
 
+@[simp] theorem mem_toVector : ∀ a (v : Vector α n), a ∈ v.toVector ↔ a ∈ v.toList :=
+  fun _ ⟨_, _⟩ => Vector.mem_mk.trans Array.mem_toArray
```

Comment:
I guess arrays have no membership?

Reply:
Uh, arrays have membership, but List.Vectors don't seem to.